### PR TITLE
RPI Lab2

### PR DESCRIPTION
**Pull request by AlexandrKolyadich**

### CODE REVIEW

File: dds.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Sanya</title>
+		<link rel="stylesheet" href="ddsa.css"> 
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,500">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+	</head>
+	<body>
+		<header>
+			<nav class ="menu">
+				<div> <a> PSD WEBSITES </a> </div>
```
Comment (AlexandrKolyadich) [inline dds.html]: ul > li > a (semantic html)

File: ddsa.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
+body{ 
+margin: 0 auto; 
+font-family: 'Montserrat', sans-serif; 
+} 
+header{ 
+background-color: #373c3f;
```
Comment (AlexandrKolyadich) [inline ddsa.css]: tabs

File: ddsa.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+background-color: #ebebeb; 
+color: #515151; 
+font-size: 1.2vw; 
+padding: 5em; 
+} 
+footer h2{ 
+color: #088ca5; 
+} 
+
+.text1{
```
Comment (AlexandrKolyadich) [inline ddsa.css:102]: to read: yandex bem

**Comment by aliaksandrkrasnou** [comment]
Approved, okay 👍

**Pull request closed without merging**